project-base/storefront/tailwind.config.js

```diff
@@ -14,36 +14,28 @@ module.exports = {
             xxl: em(1560),
```

Comment:
Shopsys logo is not visible
<img width="374" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/abd5548e-801e-4801-901e-d9208a35b4e0">